app/helpers/application_helper.rb

```diff
@@ -163,6 +163,12 @@ def sign_in_path
     end
   end
 
+  def rescheduling_hint(affected_event_count)
+    if affected_event_count > 0
+      "You have #{affected_event_count} scheduled events. Changing the conference hours will unschedule those scheduled outside the conference hours."
```

Comment:
I'd use pluralize for the word events, so that: 
You have 1 scheduled events => You have 1 scheduled event